frontend/src/components/StoreProfile/StoreProfile.js

```diff
@@ -28,7 +29,7 @@ const StoreProfile = ({ history }) => {
         }
     };
 
-    if (loadingParticipant) {
+    if (!participant) {
         return <Loading />;
     }
```

Comment:
Related to the previous comments: What happens if fetching the participant fails and participant stays `null` forever? It will probably show the loading indicator forever as well, causing the user to think it's still loading.